code/modules/mob/living/carbon/human/species_types/roguetown/other/aasimar.dm

```diff
@@ -43,8 +43,8 @@
 		OFFSET_NECK_F = list(0,-1), OFFSET_MOUTH_F = list(0,-1), OFFSET_PANTS_F = list(0,0), \
 		OFFSET_SHIRT_F = list(0,0), OFFSET_ARMOR_F = list(0,0), OFFSET_UNDIES_F = list(0,0), \
 		)
-	specstats = list("strength" = 0, "perception" = 1, "intelligence" = 2, "constitution" = -1, "endurance" = -1, "speed" = 1, "fortune" = 1)
-	specstats_f = list("strength" = 0, "perception" = 1, "intelligence" = 2, "constitution" = -1, "endurance" = -1, "speed" = 1, "fortune" = 1)
+	specstats = list("strength" = 0, "perception" = 0, "intelligence" = 2, "constitution" = -2, "endurance" = 0, "speed" = 0, "fortune" = 2)
+	specstats_f = list("strength" = 0, "perception" = 0, "intelligence" = 2, "constitution" = -2, "endurance" = 0, "speed" = 0, "fortune" = 2)
```

Comment:
forgot in review. Aasimar are supposed to be overtuned and better than other races in every way other than strength and endurance, and should be much frailer. You made them, equal to humans with a random - to con. buff them again